packages/mdc-list/index.js

```diff
@@ -234,6 +229,26 @@ class MDCList extends MDCComponent {
           listItem.click();
         }
       },
+      toggleCheckbox: (index) => {
+        const listItem = this.listElements[index];
+        const elementsToToggle =
+          [].slice.call(listItem.querySelectorAll('input[type="radio"], input[type="checkbox"]'));
+        elementsToToggle.forEach((element) => {
+          const event = document.createEvent('Event');
+          event.initEvent('change', false, true);
+
+          if (!element.checked) {
```

Comment:
I think this can be simplified to `if (!element.checked || element.type !== 'radio')` and avoid needing the else and repeating the 2 lines inside? (Move the comment up)